src/main.rs

```diff
@@ -45,7 +45,7 @@ fn main() {
         ("init",   Some(matches)) => handle_init   (matches, &mut hermit, &mut file_operations),
         ("nuke",   Some(matches)) => handle_nuke   (matches, &mut hermit, &mut file_operations),
         ("status", Some(matches)) => handle_status (matches, &mut hermit, &mut file_operations),
-        ("use",    Some(matches)) => handle_use    (matches, &mut hermit, &mut file_operations),
+        ("inhabit",    Some(matches)) => handle_inhabit    (matches, &mut hermit, &mut file_operations),
```

Comment:
I'm going to move all of the `Some(matches)` back by 1 space so that `,Some(matches)` and `handle_inhabit(matches,...` doesn't happen. Is that alright?